optuna/integration/allennlp.py

```diff
@@ -0,0 +1,103 @@
+import json
+import os
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Union
+
+import optuna
+from optuna._experimental import experimental
+
+try:
+    import _jsonnet
+    import allennlp.commands
+    import allennlp.common.util
+
+    _available = True
+except ImportError as e:
+    _import_error = e
+    _available = False
+    TrackerCallback = object
+
+
+@experimental("1.4.0")
```

Comment:
I made `AllenNLPExecutor` experimental since the major release of `AllenNLP` will come soon.
There are a lot of updates in their usages and we could face the necessity of changing the interface.
For more information, please see https://github.com/allenai/allennlp/releases/tag/v1.0-prerelease.